bin/init-lts-line

```diff
@@ -27,7 +27,7 @@ git commit -m "Towards ${baseline}.1"
 # Create dedicated packaging branch
 packaging="$(mktemp -d /tmp/jenkins-lts-rc-packaging-XXXXXXXX)"
 trap "rm -rf '$packaging'" EXIT
-git clone git@github.com:jenkinsci/packaging.git "$packaging"
+git clone https://github.com/jenkinsci/packaging.git "$packaging"
 pushd "$packaging"
 git checkout -b "$branch_name" master
 git push -u origin "$branch_name"
```

Comment:
Changing the protocol used to clone the repository will break the `git push` that happens on line 33 unless a personal access token credential is used or the remote URL is redefined to use ssh before the `git push`.
```suggestion
git clone https://github.com/jenkinsci/packaging.git "$packaging"
pushd "$packaging"
git checkout -b "$branch_name" master
git push -u origin "$branch_name"
```

Reply:
Yeah, I had a hard time to initiate the new LTS branch, because I've no SSH key setup for GitHub account on my machine at the moment.
I figured, using git or gh cli may be easier to do that in the future 🤔